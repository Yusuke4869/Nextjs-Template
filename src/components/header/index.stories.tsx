import { ComponentMeta, ComponentStoryObj } from "@storybook/react";

import Header from ".";

type T = typeof Header;
type Story = ComponentStoryObj<T>;

const data = {};

export default {
  component: Header,
  args: { ...data },
  argTypes: {},
} as ComponentMeta<T>;

export const Default: Story = {};
