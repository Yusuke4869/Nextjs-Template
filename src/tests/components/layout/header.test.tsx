import { composeStories } from "@storybook/testing-react";
import { render } from "@testing-library/react";
import * as stories from "~/components/layout/header/header.stories";

const { Default } = composeStories(stories);

describe("(components) header", () => {
  test("Snap Shot", () => {
    const { container } = render(<Default />);
    expect(container).toMatchSnapshot();
  });
});
