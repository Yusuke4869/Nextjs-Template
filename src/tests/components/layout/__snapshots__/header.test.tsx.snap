// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(components) header Snap Shot 1`] = `
<div>
  <header>
    <p>
      Header
    </p>
  </header>
</div>
`;
